lang/localize-core-element.js

```diff
@@ -0,0 +1,71 @@
+import { LocalizeMixin } from '../mixins/localize-mixin.js';
+
+export const LocalizeCoreElement = superclass => class extends LocalizeMixin(superclass) {
+
+	static async getLocalizeResources(langs) {
+		let translations;
+		for await (const lang of langs) {
+			switch (lang) {
+				case 'ar':
+					translations = await import('./ar.js');
```

Comment:
Does the delayed imports mean we need to do something special in BSI?

Reply:
I'm really, really hoping not. There's a bunch of lang file copying in the BSI build that I don't want to be necessary. I _think_ the fact that we're explicitly listing these all out means the build system can find them.

Reply:
I also hope not, although I'm not sure the specific reasons the ones in BSI are needed.

Reply:
I specifically [added this note](https://github.com/Brightspace/brightspace-integration/blob/master/rollup/rollup-wc.config.js#L50) to the new stuff to try and stop the bleeding.

Reply:
I'll test this with my local BSI and LMS. If my LMS is able to load the components and they look fine/have proper lang terms in a couple of languages, is it fine or is there something else I should be watching for?

Reply:
That should do it. Even just building BSI and then looking in the `build` directory to make sure the files exist should be enough.

Reply:
Cool, seems fine. Files are present in the build, more/less loads as expected in discussions for en and ar